app.py

```diff
@@ -0,0 +1,95 @@
+from flask import Flask, request, send_file
+from flask_limiter import Limiter
+from PIL import Image, ImageOps
+import numpy as np
+
+import tensorflow as tf
+import tensorflow.keras
+
+import numpy as np
+import PIL.Image
+import io
+import tensorflow_hub as hub
+
+hub_module = hub.load('https://tfhub.dev/google/magenta/arbitrary-image-stylization-v1-256/2')
+
+def load_img(img):
+  byteIO = io.BytesIO()
+  img.save(byteIO, format='PNG')
+  byteArr = byteIO.getvalue()
+
+  img = tf.image.decode_image(byteArr, channels=3)
+  img = tf.image.convert_image_dtype(img, tf.float32)
+
+  shape = tf.cast(tf.shape(img)[:-1], tf.float32)
+  max_dim = 512
+  long_dim = max(shape)
+  scale = max_dim / long_dim
+
+  new_shape = tf.cast(shape * scale, tf.int32)
+
+  img = tf.image.resize(img, new_shape)
+  img = img[tf.newaxis, :]
+  return img
+
+def tensor_to_image(tensor):
+  tensor = tensor*255
+  tensor = np.array(tensor, dtype=np.uint8)
+
+  if np.ndim(tensor)>3:
```

Comment:
if np.ndim(tensor) > 3: would be better, init?

Reply:
Done!